app/pages/project/floating-ips/FloatingIpsPage.tsx

```diff
@@ -122,6 +123,14 @@ export function FloatingIpsPage() {
           },
         }
     return [
+      {
+        label: 'Edit',
+        onActivate: () => {
+          navigate(pb.floatingIpEdit({ project, floatingIp: floatingIp.name }), {
+            state: floatingIp,
+          })
+        },
+      },
```

Comment:
I don't think this `state` bit is meant to be in there, right?

Also: I had forgotten about this trick, but it's probably worth doing. I bet we've forgotten to do it on other edit forms. By taking the `floatingIp` object we already have on hand and putting it in RQ's cache, we can make the form open instantly because the loader's data is already there.

https://github.com/oxidecomputer/console/blob/3ad46af4f1923bbee040cea798d608dc4c059302/app/pages/ProjectsPage.tsx#L68-L77

Reply:
Ah, that's nice.